README.adoc

```diff
@@ -45,6 +45,9 @@ plugins{
 
 jsonschema2pojo {
     targetPackage = 'org.example.api' // specify package for your needs
+    
+    source.setFrom files("${sourceSets.main.output.resourcesDir}/json/schema")
```

Comment:
In the samples we use this input directory `source.setFrom files("${project.rootDir}/schema")`
Yes it's missing, but why use this _output_ directory ?

Reply:
Source folder by default is located under `main/resources/json` and taken after resources are processed. This is the reason this folder has been suggested. 

@pgalbraith  If you want to specify this folder manually, it's better not to use `output` as it will be never be generated in that case and task will silently fail. 

@hannesa2 should we raise a warning in case no files are given for a task?

Reply:
@eirnym 👍

Reply:
I just used the example set from the original plugin (https://github.com/joelittlejohn/jsonschema2pojo/tree/master/jsonschema2pojo-gradle-plugin#usage), but any example set is fine with me.